api/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import { app } from "@azure/functions";
+import { CreateProduct } from "./functions/CreateProduct";
+import { DeleteProduct } from "./functions/DeleteProduct";
+import { UpdateProduct } from "./functions/UpdateProduct";
+
+app.http('CreateProduct',{
+    methods: ['GET', 'POST'],
+    authLevel: 'anonymous',
+    handler: CreateProduct
+})
```

Comment:
Please add a semicolon and linting/esling to the package.json to catch this.

Reply:
Done